tests/comm/packet_check.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Created by Heather D'Souza on 2017-11-29.
+//
+
+#include "comm/PacketConstructor.h"
+#include "../../include/gtest-1.8.0/include/gtest/gtest.h"
+
+using namespace wlp;
+
+class TestPacket : public PacketConstructor {
+public:
+    TestPacket(const float* p_data, const Bitset<2> &packetType, const Bitset<6> &packetName) : PacketConstructor(p_data, packetType, packetName) {}
+
+    void getFromParent(float *data) {
```

Comment:
I'm not 100% sure why we need this virtual function instead of just directly calling `get`. Could you provide a use case in which we might need multiple subclasses that do different things here? (Other than modifiy the provided array, which could be done outside the class).

Also the default implementation of the virtual function could just be this, which means you wouldn't need to subclass `PacketConstructor` to test it.